language/tools/move-explain/src/main.rs

```diff
@@ -17,6 +18,9 @@ struct Args {
     /// The abort code returned with a `MoveAbort` error
     #[structopt(long = "abort-code", short = "a")]
     abort_code: u64,
+    /// Path to the errmap file.
```

Comment:
I had to `grep` around the project to find out what an "errmap" was (at first I guessed it could be a Rust thing or a crate), and the best explanation I found was in `language/diem-framework/README.md` -- so, presumably not a file that's included in the "Move part" of this project.

I don't have a great suggestion here, though. You could provide a default value for this argument so users can easily find an example of an `.errmap` file, but that changes the behavior of `move-explain` in a way you probably don't want... so, I don't know. Maybe just create a TODO to document the format of `.errmap` files or how to generate them, or add a README to the `move-errmapgen` crate?

Reply:
I made this comment slightly more explicit ("errmap file" => "error code mapping file"). I found it a little bit weird to add a TODO in this PR, but I filed an issue for this: #8642.